src/components/print/PrintDirective.js

```diff
@@ -272,42 +272,41 @@ goog.require('ga_urlutils_service');
           return enc;
         },
 
+        'matrixIds': function(tilegrid, extent) {
+          var matrixIds = [];
+          var ids = tilegrid.getMatrixIds();
+          var resolutions = tilegrid.getResolutions();
+          angular.forEach(resolutions, function(value, key) {
+            var resolution = parseFloat(value);
+            var z = tilegrid.getZForResolution(resolution);
+            var tileSize = tilegrid.getTileSize(z);
+            var topLeftCorner = tilegrid.getOrigin(z);
+            var minX = topLeftCorner[0];
+            var maxY = topLeftCorner[1];
+            var maxX = extent[2] || 900000;
```

Comment:
could we avoid hardcoded value ? You can have this value from the projection extent or the default extent in gaMapUtils.